languages/go/exercises/concept/comments/.docs/introduction.md

```diff
@@ -0,0 +1,53 @@
+In the previous exercise, we saw that there are two ways to write comments in Go: single-line comments that are preceded by `//`, and multiline comment blocks that are wrapped with `/*` and `*/`.
+
+## Documentation comments
+
+In Go, comments play an important role in documenting code. They are used by the `godoc` command, which extracts these comments to create documentation about Go packages. A documentation comment should be a complete sentence that starts with the name of the thing being described and ends with a period.
+
+Comments should precede packages as well as exported identifiers, for example exported functions, methods, package variables, constants, and structs, which you will learn more about in the next exercises.
+
+A package-level variable can look like this:
+
+```go
+// TemperatureFahrenheit gives a certain temperature in degrees Fahrenheit.
+var TemperatureFahrenheit float64
+```
+
+## Package comments
+
+Package comments should be written directly before a package clause (`package x`) and begin with `Package X ...` like this:
+
+```go
+// Package kelvin provides tools to convert
+// temperatures to and from Kelvin.
+package kelvin
+```
+
+## Function comments
+
+A function comment should be written directly before the function declaration. It should be a full sentence that starts with the function name. For example, an exported comment for `function Test` should take the form `Test ...`. It should also explain what arguments the function takes, what it does with them, and what its return values mean, ending in a period):
+
+```go
+// CelsiusFreezingTemp returns an integer value equal to the temperature at which water freezes in degrees Celsius.
+func CelsiusFreezingTemp() int {
+	return 0
+}
+```
+
+## Golint
```

Comment:
I'd remove the entire golint block from the intro... or maybe just mention it and remove the installation guide. What do you think?

Reply:
I'd vote for keeping this. Most Gophers probably would never write documentation comments if it weren't for `golint`. Indeed, I never give any feedback on Exercism solutions until they pass `golint`, so I'd say it's pretty critical.

Reply:
Yes, I also vote to keep it because it's also a good opportunity to learn more about `$PATH`. For me, installing `golint` was the thing that prompted me to look up the relationship between `$PATH` and Go.

Reply:
We are keeping it, but in the `after.md`. It does not belong in the `introduction.md`. If you want we can mention it there but the installation instructions should be removed.

Here the guideline I am basing this on:  https://exercism.github.io/v3/#/docs/concept-exercises?id=documentation-files. It says explicetely (among other things): `Only information that is needed to understand the fundamentals of the concept and solve the exercise should be provided. Extra information should be left for the after.md.`

Reply:
Note that in any concept exercise there will be an analyzer checking the code against `golint`. If it does not conform we will block the student and let them know their code does not follow the style golint enforces. I think that should help with mentoring later on.

Reply:
One last note: This is one of the first exercises. We should assume that most students will solve this in the browser without even installing Go itself, since v3 will have full browser support, automatically run tests and analyse the solution. We assume that many will start learning a language in the browser and see how they like it before they commit to it and install it. By then they will have progressed beyond this exercise and all they have will be a history of the `after.md` files to go back and review.